deps/llvm.mk

```diff
@@ -2,7 +2,7 @@
 include $(SRCDIR)/llvm-ver.make
 
 ifneq ($(USE_BINARYBUILDER_LLVM), 1)
-LLVM_GIT_URL_BASE ?= http://llvm.org/git
+LLVM_GIT_URL_BASE ?= https://llvm.org/git
```

Comment:
I don't know what the origin URL contain, maybe has all LLVM's project or only LLVM.
+ `https://llvm.org/git` => `https://github.com/llvm/llvm-project.git` (All project/official )
  or `https://github.com/llvm-mirror/llvm.git` (Unofficial / LLVM only and Other project [Unofficial Automated Mirror of LLVM](https://github.com/llvm-mirror))